models/user.js

```diff
@@ -454,7 +455,7 @@ UserService.verifyToken = function (token) {
      * and make an entry if it does not exist.
      * reject if entry already exists.
      */
-    return this.findOne({id: decoded.userId});
+    return this.findById(decoded.userId);
```

Comment:
Should be UserService.findById, and these service functions should not use the function keyword, prefer use of arrow functions.